lib/auth/apiserver.go

```diff
@@ -1123,6 +1123,53 @@ func (s *APIServer) getClusterCACert(auth ClientI, w http.ResponseWriter, r *htt
 	return localCA, nil
 }
 
+func (s *APIServer) rotateUserTokenSecrets(auth ClientI, w http.ResponseWriter, r *http.Request, p httprouter.Params, version string) (interface{}, error) {
```

Comment:
All new APIs should be GRPC not HTTP/JSON which has been deprecated. Take a look at https://github.com/gravitational/teleport/blob/master/lib/auth/grpcserver.go.